server/app/cloud.go

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package app
+
+import (
+	"errors"
+
+	"github.com/mattermost/focalboard/server/model"
+)
+
+var ErrNilPluginAPI = errors.New("server not running in plugin mode")
+
+func (a *App) GetBoardsCloudLimits() (*model.BoardsCloudLimits, error) {
+	if a.pluginAPI == nil {
+		return nil, ErrNilPluginAPI
+	}
```

Comment:
It depends on the UI implementation, but should this just return an unlimited `BoardCloudLimits` object, so that the UI code works the same.

Reply:
0/5 on this, I've updated the response to return a `Not Implemented` to follow what Channels is doing for unlicensed and non-cloud instances, but I'm happy to return an empty object. `Limit = 0` should effectively be the same as no limit, so we can make no distinction and do it like that